src/components/text-editor/text-editor.style.ts

```diff
@@ -70,15 +71,19 @@ const StyledEditorOutline = styled.div<{
   isFocused?: boolean;
   hasError?: boolean;
 }>`
-  ${({ isFocused, hasError }) => css`
+  ${({ isFocused, hasError, theme }) => css`
     border-radius: var(--borderRadius050);
     outline: none;
     min-width: 320px;
 
     ${isFocused &&
     css`
-      outline: 3px solid var(--colorsSemanticFocus500);
-      outline-offset: ${hasError ? "2px;" : "1px;"};
+      ${!theme.focusRedesignOptOut
+        ? addFocusStyling()
+        : `
+        outline: 3px solid var(--colorsSemanticFocus500);
```

Comment:
**comment** (non-blocking): as already commented in a few other places, it seems inconsistent that these old focus styles aren't extracted into a const when they are in most other components.